codes/pthread_macros/pthread.m4.POSIX_BARRIER

```diff
@@ -0,0 +1,98 @@
+dnl Found on the web.  Written by Bastiaan Stougie, October 2003
+dnl Modified by Alberto Ros and Christos Sakalis, 2015
```

Comment:
That's ok, no need for attribution. I simply used the same thing found in the POSIX_BARRIER file from the null macros,